app/controllers/Deliveries.scala

```diff
@@ -1,22 +1,41 @@
 package controllers
 
 import com.nashtech.delivery.v1.controllers.DeliveriesController
-import play.api.mvc.{AnyContent, ControllerComponents, Request}
+import database.PostgresConnection
+import play.api.mvc.{AbstractController, AnyContent, ControllerComponents, Request}
 import service.DeliveriesService
 
 import javax.inject.Inject
 import scala.concurrent.Future
 
 class Deliveries @Inject() (
   deliveriesService: DeliveriesService,
-  val controllerComponents: ControllerComponents,
-) extends DeliveriesController {
+  controllerComponents: ControllerComponents,
+  postgresDAO: PostgresConnection
+) extends   AbstractController(controllerComponents)  with DeliveriesController {
   override def getById(request: Request[AnyContent], merchantId: String, id: String): Future[GetById] = {
     Future.successful(
       deliveriesService.getByid(merchantId, id) match {
         case Left(_) => GetById.HTTP404
-        case Right(delivery) => GetById.HTTP200(delivery)
+        case Right(delivery) =>
+          index
```

Comment:
Seems like this was used for testing purpose and should be removed now.

Reply:
Yes, will remove it.